studio/components/interfaces/Home/NewProjectPanel/NewProjectPanel.tsx

```diff
@@ -1,28 +1,246 @@
-import { DisplayApiSettings, DisplayConfigSettings } from 'components/ui/ProjectSettings'
+import Link from 'next/link'
 import { FC } from 'react'
-import GetStartedPanel from './GetStartedPanel'
+import SVG from 'react-inlinesvg'
+import { useRouter } from 'next/router'
+import { Button, IconKey, IconArchive, IconExternalLink, IconCode } from '@supabase/ui'
+
+import Panel from 'components/ui/Panel'
+import APIKeys from './APIKeys'
+import GetStartedHero from './GetStartedHero'
 
 interface Props {}
 
-const NewProjectPanel: FC<Props> = ({}) => (
-  <div className="grid grid-cols-12 gap-8">
-    <div className="col-span-12">
-      <GetStartedPanel />
-    </div>
-    <div className="col-span-4">
-      <div className="space-y-2">
-        <h3 className="text-xl text-scale-1200">Connecting to your new project</h3>
+const NewProjectPanel: FC<Props> = ({}) => {
+  const router = useRouter()
+  const { ref } = router.query
+
+  return (
+    <div className="grid grid-cols-12 gap-20">
+      <div className="col-span-12">
+        <div className="flex flex-col space-y-20">
+          <div className="flex flex-col justify-between h-full">
+            <div className="space-y-2">
+              <h3 className="text-xl text-scale-1200">Welcome to your new project</h3>
+              <p className="text-base text-scale-1100">
+                Your project has been deployed on its own instance, with its own API all set up and
+                ready to use.
+              </p>
+            </div>
+          </div>
+
+          <div className="grid grid-cols-12 flex gap-4">
+            <div className="col-span-7 space-y-8 flex flex-col justify-center">
+              <div className="space-y-2">
+                <h3 className="text-xl text-scale-1200">
+                  Get started by building out your database
```

Comment:
Feels like something might be needed between these sections?

<img width="1296" alt="CleanShot 2022-09-23 at 10 00 30@2x" src="https://user-images.githubusercontent.com/105593/191960648-c1ae1763-2dee-41de-9e25-779bc0f63aee.png">


An <hr> maybe? Not sure
<img width="1287" alt="CleanShot 2022-09-23 at 10 01 21@2x" src="https://user-images.githubusercontent.com/105593/191960712-1b0d92c5-73df-461a-8cfd-c5d428e4ad2f.png">

Reply:
think this should be okay - was using stripe's docs as one of my references https://stripe.com/docs